### DISCUSSION TITLE
Guidance how to mimic the automatic fan settings from the Adrenaline to FanControl.ADLX

### DISCUSSION THREAD

**Discussion by godzinilla** [Rem0o/FanControl.ADLX#46] [General]
Hello all.

I am asking mainly the @Rem0o but any help will be appreciated here :) From my observations I found that the automatic fan settings provided by the Adrenaline software works better than my custom settings in FanControl so I want to mimic these settings to the FanControl as a good starting point for customization. Is there a way to read somewhere how the Adrenaline sets the fan speeds? We talking about setting the fans to automatic, without "0 RPM" and with max speed set to 60%.

Thank you.

**Comment by Rem0o** [comment]
Easy, either leave the toggle off in fancontrol so it keeps the default behaviour, else use a graph.

**Reply by godzinilla** [reply]
So there is no easy readout for the Adrenaline settings set this way? Hoped that it will be somewhere stored in the system or read by the FanControl itself.
